src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -15,16 +15,54 @@ Add `AddAWSLambdaConfigurations()` to `TracerProvider`.
 ```csharp
 TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
         // add other instrumentations
-        .AddAWSLambdaConfigurations()
+        .AddAWSLambdaConfigurations(options => options.DisableAwsXRayContextExtraction = true)
```

Comment:
Good idea. Added it to the default example.

Reply:
This was not meant as suggestion, but I am wondering why we show that option in the example. If the option would be good as default-on for some reason, probably we should change it in the implementation, otherwise I would recommend leaving the example as it was before with just

```suggestion
        .AddAWSLambdaConfigurations()
```

And maybe show a call with customized options in the section below.

Reply:
But may be before we change implementation it's worth to keep DisableAwsXRayContextExtraction in the example ? Otherwise it may lead to some misunderstandings.

Reply:
In general looks good, I am not AWSLamda expert. If `DisableAwsXRayContextExtraction` set to `true` should be default option lets keep it on both examples. Otherwise remove it.

Reply:
Then lets keep it in both examples as is.